src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
 #include "I18N.hpp"
 
 const I18N::Text I18N::all_texts_french[] = {
-    { I18N_BUTTON_YES, "Yes" },
-    { I18N_BUTTON_NO, "No" },
+    { I18N_BUTTON_YES, "Oui" },
+    { I18N_BUTTON_NO, "Non" },
     { I18N_BUTTON_OK, "Ok" },
-    { I18N_BUTTON_CANCEL, "Cancel" },
-    { I18N_BUTTON_CLOSE, "Close" },
-    { I18N_BUTTON_START_SERVER, "Start Server" },
-    { I18N_BUTTON_DELETE, "Delete" },
+    { I18N_BUTTON_CANCEL, "Annuler" },
+    { I18N_BUTTON_CLOSE, "Fermer" },
+    { I18N_BUTTON_START_SERVER, "Démarrer le Serveur" },
+    { I18N_BUTTON_DELETE, "Supprimer" },
 
-    { I18N_TILETYPE01, "non blocking" },
-    { I18N_TILETYPE02, "blocking" },
-    { I18N_TILETYPE03, "falling only blocking" },
-    { I18N_TILETYPE04, "red base" },
-    { I18N_TILETYPE05, "blue base" },
-    { I18N_TILETYPE06, "killing" },
+    { I18N_TILETYPE01, "non bloquant" },
+    { I18N_TILETYPE02, "bloquant" },
+    { I18N_TILETYPE03, "bloquant juste la tombée" },
+    { I18N_TILETYPE04, "base rouge" },
+    { I18N_TILETYPE05, "base bleue" },
+    { I18N_TILETYPE06, "mortel" },
     { I18N_TILETYPE07, "jump, no descend" },
-    { I18N_TILETYPE08, "speed race finish" },
-    { I18N_TILETYPE09, "hill zone" },
+    { I18N_TILETYPE08, "fin de course" },
+    { I18N_TILETYPE09, "zone de colline" },
 
     { I18N_WINTITLE_GAME, "Goat Attack" },
-    { I18N_WINTITLE_EDITOR, "Goat Attack Map Editor" },
+    { I18N_WINTITLE_EDITOR, "Éditeur de cartes" },
 
-    { I18N_SUBSYSTEM, "working with ${1}" },
-    { I18N_MODE_IS_NORMAL, "mode is normal game" },
-    { I18N_MODE_IS_DEDICATED, "mode is dedicated server" },
+    { I18N_SUBSYSTEM, "fonctionne avec ${1}" },
+    { I18N_MODE_IS_NORMAL, "mode 'jeu normal'" },
+    { I18N_MODE_IS_DEDICATED, "mode 'serveur dédié'" },
 
-    { I18N_MAINMENU_PLAY, "Play" },
-    { I18N_MAINMENU_LAN, "Create Local LAN Server" },
-    { I18N_MAINMENU_PACKAGES, "List Loaded Packages" },
-    { I18N_MAINMENU_OPTIONS, "Options And Settings" },
-    { I18N_MAINMENU_CREDITS, "Credits" },
-    { I18N_MAINMENU_QUIT, "Quit Game" },
-    { I18N_MAINMENU_VERSION, "Current Version: ${1}" },
+    { I18N_MAINMENU_PLAY, "Jouer" },
+    { I18N_MAINMENU_LAN, "Créer un serveur local" },
+    { I18N_MAINMENU_PACKAGES, "Lister les paquets chargés" },
+    { I18N_MAINMENU_OPTIONS, "Options et Préférences" },
+    { I18N_MAINMENU_CREDITS, "Crédits" },
+    { I18N_MAINMENU_QUIT, "Quitter" },
+    { I18N_MAINMENU_VERSION, "Version actuelle: ${1}" },
```

Comment:
In theory we should have a space before ":" in French. It's not necessarily a big loss if we don't put it, but it might seem more professional with it.